plugins/chat/lib/chat_message_mentions.rb

```diff
@@ -67,7 +79,6 @@ def chat_users
       .joins("LEFT OUTER JOIN user_chat_channel_memberships uccm ON uccm.user_id = users.id")
       .joins(:user_option)
       .real
-      .not_suspended
```

Comment:
I removed it here because if user is suspended, we still want to create a mention for him (and it added in other places because we don't want to create a notification for him). You can see this change is in a dedicated commit – 1c2e109e.